section2-the-ruby-programming-language/authenticator/authenticator.rb

```diff
@@ -0,0 +1,45 @@
+# prompts for username and password
+# you can attempt for max 3 times
+
+# create hash using symbol as keys
+users = [
+  { username: "Jack", password: "beanstalk"},
```

Comment:
Consistency pls.
You have space after the `{` but no space before `}`. Choose one way and stick with it